include/gatekeeper_gt.h

```diff
@@ -51,6 +58,15 @@ struct gt_instance {
 
 	/* The lua state that belongs to the instance. */
 	lua_State     *lua_state;
+
+	/*
+	 * The fragment table maintains information about already
+	 * received fragments of the packet.
+	 */
+	struct rte_ip_frag_tbl *frag_tbl;
+
+	/* The mbuf death row contains packets to be freed. */
+	struct rte_ip_frag_death_row death_row;
```

Comment:
Is there a reason why we prefer putting `death_row` here, instead of on the stack at the beginning of `gt_proc()`?